tests/integration/models/reddit/test_comment.py

```diff
@@ -182,6 +182,21 @@ def test_refresh__removed_comment(self):
         assert ('This comment does not appear to be in the '
                 'comment tree',) == excinfo.value.args
 
+    def test_refresh__with_reply_sort_and_limit(self):
+        with self.recorder.use_cassette('TestComment.test_refresh__with_reply_sort_and_limit'):
```

Comment:
The build is failing because this line is longer than our styleguide permits:

    ./tests/integration/models/reddit/test_comment.py:186:80: E501 line too long (95 > 79 characters)

I think once that's resolved, all the tests should pass then we can merge this PR in. Thanks!